src/main/jbake/content/blog/2017/05/31/hawkular-grafana-ootb.adoc

```diff
@@ -0,0 +1,253 @@
+<2> Using the official Hawkular Services image. Accessible on http://localhost:8080/hawkular/metrics or http://localhost:8080/hawkular/alerts, with credentials _jdoe_/_password_.
+<3> The volume is commented-out, so data is not persisted if you run `docker-compose down`. To persist data, uncomment these lines and eventually set something different than `/tmp/opt/hawkular/server` as the host directory. Make sure this directory exists on your host.
+<4> The Hawkular agent is disabled by default, which means Hawkular is not self-monitored. You can enable it if you want, and use the `hawkular-self` dashboard.
+<5> The Cassandra node. Note that it's not exposed outside of the compose's subnetwork.
+<6> The Grafana server with pre-installed Hawkular plugin, accessible on http://localhost:3000, with credentials _admin_/_admin_
+<7> This image contains a bootstrap script that configures a Grafana datasource with the given tenant (<8>), which is also injected in dashboard templates defined here (<9>). Those dashboards are sent to Grafana through its REST API. Current available dashboard templates are listed below.
+
+If you want to have more than 1 tenant configured, you can add other sections to this compose file (_bootstrap2_, _bootstrap3_, etc. the name doesn't matter),
+with the same docker image and different environment variables. You are limited, however, in the number of tenants per dashboard (a single dashboard cannot use several tenants).
+You need to duplicate dashboards, or create them manually.
+
+== Available dashboard templates
+
+Only a few are available right now, but the list should grow and pull requests are welcome!
+
+Some dashboards may be very specific, dedicated to the demos presented here, while others aim to be generic and reusable. I guess the next step will be to inject custom dashboards through volume in docker-compose without having the rebuild the bootstrap image. Anyway, the current list is:
+
+- *vertx-hwk*: for Vert.X applications that use the link:http://vertx.io/docs/vertx-hawkular-metrics/java/[vertx-hawkular-metrics] module.
+- *vertx-dw*: for Vert.X applications that use the link:http://vertx.io/docs/vertx-dropwizard-metrics/java/[vertx-dropwizard-metrics] module along with the link:https://github.com/hawkular/hawkular-metrics/tree/master/clients/dropwizard[Hawkular reporter].
+- *hawkular-self*: for Hawkular self monitoring using its WildFly agent (but could work with other WF apps)
+- *cpudemo*: for the CPU demo (explained below)
+- *falco*: for the game/demo "link:https://github.com/jotak/falco-demo[Falco the Hawk]"
+
+== CPU Demo
+
+Want a demo? Take the docker-compose file located in `cpudemo/` (link:https://github.com/jotak/hawkular-grafana-outofthebox/blob/master/cpudemo/docker-compose.yml[yes, this one]) and store it locally. From its location run:
+
+[source,bash]
+----
+docker-compose up
+----
+You may have to wait several minutes the first time you run it so that the docker engine downloads everything.
```

Comment:
In this paragraph you use passive voice in the second sentence but the first. I would prefer the passive voice in the firs sentence too.